pyvrp/_CostEvaluator.pyi

```diff
@@ -0,0 +1,26 @@
+from typing import overload
```

Comment:
Can we add this class to the docs? Something like
```sphinx
.. automodule:: pyvrp._CostEvaluator
   
   .. autoapiclass:: CostEvaluator
      :members:
```
in `docs/source/api/pyvrp.rst`.